cut max-frequency to half because at full speed mmc is buggy

--- a/arch/arm64/boot/dts/mediatek/mt8173-elm.dtsi	2020-12-28 21:22:16.614538427 +0000
+++ b/arch/arm64/boot/dts/mediatek/mt8173-elm.dtsi	2020-12-28 21:23:59.702157398 +0000
@@ -381,7 +381,7 @@
 	pinctrl-0 = <&mmc1_pins_default>;
 	pinctrl-1 = <&mmc1_pins_uhs>;
 	bus-width = <4>;
-	max-frequency = <200000000>;
+	max-frequency = <100000000>;
 	cap-sd-highspeed;
 	sd-uhs-sdr50;
 	sd-uhs-sdr104;
